sci-mathematics/rstudio/rstudio-1.4.1106.ebuild

```
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake qmake-utils
+
+QT_VER=5.12.6
+QT_SLOT=5
+
+DESCRIPTION="IDE for the R language"
+HOMEPAGE="
+	https://rstudio.com
+	https://github.com/rstudio/rstudio/"
+SRC_URI="
+	https://github.com/rstudio/rstudio/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries.zip -> ${P}-core-dictionaries.zip
+"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="-headless test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-db/postgresql:*
+	dev-db/sqlite
+	>=dev-lang/R-3.0.1
+	>=dev-libs/boost-1.69:=
+	>=dev-libs/mathjax-2.7
+	dev-libs/soci
+	sys-process/lsof
+	!headless? (
+		>=dev-qt/qtcore-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtdbus-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtdeclarative-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtnetwork-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtopengl-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtpositioning-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtsensors-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtsql-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtsvg-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtwebchannel-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtwebengine-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtxml-${QT_VER}:${QT_SLOT}
+		>=dev-qt/qtxmlpatterns-${QT_VER}:${QT_SLOT}
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	app-arch/unzip
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.4.1106-add-missing-boost-namespace-import.patch"
+	"${FILESDIR}/${PN}-1.4.1106-add-missing-boost-bind-import.patch"
+	"${FILESDIR}/${PN}-1.4.1106-fix-server-install-paths.patch"
+	"${FILESDIR}/${PN}-1.4.1106-remove-gwt-build.patch"
+	"${FILESDIR}/${PN}-1.4.1106-remove-hardcoded-internal-mathjax-and-pandoc-deps.patch"
+	"${FILESDIR}/${PN}-1.4.1106-remove-mathjax-internal-dep-check.patch"
+	"${FILESDIR}/${PN}-1.4.1106-soci-cmake-find_library.patch"
+)
+
+src_unpack() {
+	local dict_archive_name="${P}-core-dictionaries.zip"
+
+	for ARCHIVE in ${A}
```

This is fine, although unusual style for ebuilds.

Normally:
```suggestion
	for ARCHIVE in ${A} ; then
```
or
```suggestion
	for ARCHIVE in ${A}; then
```

---

Yes, I noticed the preference for the `; then` on the same line - why is that? I think it looks cleaner and is easier to read with `then` on its own line, plus it gets rid of the ugly `;`.

I can change it if absolutely necessary.

---

I suppose it should be okay. I think there's value in using the same-ish style as everybody else as it eases readability though.

---

per `#gentoo-dev` convo, I went ahead and just changed it.